ISPN-13050 Gustavo's encoding doc review

oraNod opened the pull request:
https://issues.redhat.com/browse/ISPN-13050

diegolovison commented:
add to whitelist

danberindei commented on documentation/src/main/asciidoc/topics/con_cache_encoding.adoc:
```diff
@@ -6,19 +6,21 @@ Encoding is the format, identified by a media type, that {brandname} uses to sto
+
+.Use ProtoStream encoding
+
+{brandname} Server cannot convert data to and from certain media types and generates an error when client requests include a media type that is not supported.
```
I would focus on request+cache media type pairs, because whether the server can process a request or not depends on both. E.g.

```suggestion
{brandname} Server returns an error when the client request includes a media type that cannot be converted to or from the cache encoding's media type.
```

gustavonalle commented on documentation/src/main/asciidoc/topics/con_cache_encoding.adoc:
```diff
@@ -6,19 +6,19 @@ Encoding is the format, identified by a media type, that {brandname} uses to sto
-[TIP]
-====
-Always configure an encoding for remote caches if you want to use multiple clients, such as {brandname} Console or CLI, Hot Rod, REST, as well as server-side tasks and scripts.
+If the remote cache does not have any encoding configuration, {brandname} Server stores keys and values as generic `byte[]` without any media type information.
+In this case {brandname} Server returns keys and values for any client requests as they are stored in the remote cache, without any media type conversion.
```
```suggestion
If the remote cache does not have any encoding configuration, {brandname} Server stores keys and values as generic `byte[]` without any media type information, that can lead to unexpected results when converting data to clients asking for different formats
```

gustavonalle reviewed (changes requested):
LGTM, just a single clarification needed

oraNod commented:
> LGTM, just a single clarification needed

@gustavonalle Thanks a mill for the added clarity. I've squashed and rebased. Please feel free to do the honours and merge if you approve.

danberindei commented:
Merged, thanks Don!
